.github/workflows/react.yml

```diff
@@ -24,6 +24,15 @@ jobs:
       run: yarn install
     - name: Run tests
       run: yarn test
+  
+  coverage:
+    name: Jest Coverage Report
```

Comment:
Apologies, one additional thought: Do you want to add `needs: [ unitTests ]` to be successful before running? This may save some needless coverage runs.

Reply:
Thats a good idea, and better approach than conditional check and run only for 20. proceeding with it